config/tf_modules/k8s-service/main.tf

```
@@ -8,7 +8,7 @@ terraform {
 }
 
 resource "helm_release" "k8s-service" {
-  chart = "${path.module}/../../opta-k8s-service-helm"
+  chart = "${path.module}/../../opta-k8s-service-helmz"
```

```suggestion
  chart = "${path.module}/../../opta-k8s-service-helm"
```

This looks like a typo or something that wasn't meant to be part of this PR.